src/01_bignum.py

```diff
@@ -1,4 +1,5 @@
 # Print out 2 to the 65536 power
```

Comment:
good job. you showed great understanding of python methods

Reply:
Thanks